src/expire.c

```diff
@@ -390,21 +380,16 @@ void expireSlaveKeys(void) {
             if ((dbids & 1) != 0) {
                 redisDb *db = server.db+dbid;
                 dictEntry *expire = dictFind(db->expires,keyname);
-                int expired = 0;
 
-                if (expire &&
-                    activeExpireCycleTryExpire(server.db+dbid,expire,start))
-                {
-                    expired = 1;
-                }
-
-                /* If the key was not expired in this DB, we need to set the
-                 * corresponding bit in the new bitmap we set as value.
-                 * At the end of the loop if the bitmap is zero, it means we
-                 * no longer need to keep track of this key. */
-                if (expire && !expired) {
-                    noexpire++;
-                    new_dbids |= (uint64_t)1 << dbid;
```

Comment:
what's this change? just an unnecessary refactoring? i'd rather not unless you have a reason to change that block.